payments_service/settings/base.py

```diff
@@ -110,7 +110,8 @@
 EMAIL_ALLOWED_LIST = ['.*@mozilla\.com$']
 
 # The root URL for all CSS etc.
-EMAIL_URL_ROOT = 'http://192.168.99.100:8000'
+EMAIL_URL_ROOT = (os.environ.get('EMAIL_URL_ROOT') or
+                  'http://192.168.59.103:8000')
```

Comment:
Are you using docker-machine, it defaults to http://192.168.99.100 for the first machine now. But all of these will change, so having the `os.environ` there is great.